ROCm_Tools/Doxyfile

```diff
@@ -0,0 +1,2447 @@
+## Copyright (c) 2019-2020 Advanced Micro Devices, Inc.
```

Comment:
This file is specific to the ROCdbgapi and is not a general Tools Doxygen file. So I think it belongs in a ROCdbgapi subdirectory together with the include file it references.